SECURITY.md

```diff
@@ -171,11 +171,9 @@ rules](https://bughunters.google.com/about/rules/6521337925468160/google-open-so
 
 ### Reporting vulnerabilities
 
-Please email reports about any security related issues you find to
-`security@tensorflow.org`. This mail is delivered to a small security team. For
-critical problems, you may encrypt your report (see below).
+Please fill out [this report form](https://forms.gle/mr12SgzXENhxQ7jD6) about any security related issues you find. This mail is delivered to a small security team. For critical problems, you may encrypt your report (see below).
```

Comment:
You need to remove the "this mail is delivered to a small security team" line.

Also, does the form support the PGP format at the end?

Reply:
I removed that line, and I kept the email for sensitive vulnerabilites that need to be encrypted.